catalog/app/components/Preview/renderers/IFrame.js

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { styled } from '@material-ui/styles'
+
+const IFrame = styled((props) => (
```

Comment:
Please add note on Content-Type `text/html`

Reply:
where should i put that?